### PR TITLE
Startup profile styling

### LINKED ISSUE #40: Design the startup profile page

### PR DESCRIPTION
**Pull request by Karyum**
close #40

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	border-spacing: 0;
+}
+
+/* END OF CSS RESET */
+
+body {
+  /*font-family: Helvetica, sans-serif;*/
+}
+
+.our-purple {
```
Comment (esraajb) [inline public/style.css]: bground-purple will be easier to read

File: src/views/startup_profile.hbs [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-    <a class="" href={{startup.startupWebsite}}></a>
+  <div class=" w-80 h5 absolute mt5">
+    <img class="fl ml5 mb5 h-75" src="../fac.png" alt="">
+    <p class=" w-80 f4 h1 dib absolute ml5 mt4">{{startup.startupDescription}}</p>
+  </div>
+  <div class="bottom-2 absolute h3 right-0 w-60">
+    <span class="right-2 absolute f3">Founder: {{startup.founderName}}</span>
+    <span class="fl f3">{{startup.email}}</span>
+    <br><br>
+    <a class="fl f3 mt2" href={{startup.startupWebsite}}>Link to the website</a>
```
Comment (esraajb) [inline src/views/startup_profile.hbs]: Link to the website -> 'website'

File: src/views/layouts/main.hbs [hunk 3]
```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="/style.css">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
     <title></title>
```
Comment (esraajb) [inline src/views/layouts/main.hbs]: title: Arab Innovators

File: public/style.css (hunk 1 above)
Comment (Karyum) [inline public/style.css]: i just found out that we are not using it anyway so i'll get rid of it instead

**Comment by Karyum** [comment]
screenshot from 2017-07-08 16-35-23

@Suuuuha requested a screenshot of the startup profile after that it was styled using tachyons

**Comment by esraajb** [comment]
screen shot 2017-07-08 at 4 58 21 pm

**Comment by esraajb** [comment]
@Karyum the wireframe is different

**Comment by Karyum** [comment]
screenshot from 2017-07-08 19-55-51

@Suuuuha @esraajb what do you think?

**Comment by esraajb** [comment]
Great job. Just change the order of the email and the website as its shown in the wireframe and use a capital first-letter in the sector and stage.

**Comment by Karyum** [comment]
yeah will change the order also the sector and stage are coming from the database so the capital letter thing is fixed in this pull request #37

**Pull request merged**